### DISCUSSION TITLE
singularity

### DISCUSSION THREAD

**Discussion by xiaoli-dong** [jodyphelan/TBProfiler#291] [General]
Do you have plan for create a singularity image of tbprofiler?

**Comment by jodyphelan** [comment]
Hi @xiaoli-dong 

Docker images are automatically built for every release. I think you should be able to use these with singularity.

https://quay.io/repository/biocontainers/tb-profiler?tab=tags

**Reply by xiaoli-dong** [reply]
I am building a nextflow pipeline. A lot nextflow pipelines retrieve singularity images from https://depot.galaxyproject.org/singularity/ . If you have tbprofiler image there, it can be very handy.

**Reply by jodyphelan** [reply]
Ok I'll look into this and see if I can get tb-profiler in there
